Maths/Transpose of a matrix/src/TransposeOfMatrix/MainClass.java

```diff
@@ -0,0 +1,35 @@
+package TransposeOfMatrix;
+
```

Comment:
- Please add documentation as per documentation standards.
- Please format the code as per formatting and styling standards.